src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationHandler.cs

```diff
@@ -159,14 +160,44 @@ protected override AuthenticationTicket AuthenticateCore()
             return new TokenResponse(oauth2Token);
         }
 
-        protected virtual async Task<AuthenticationTicket> GetUserInformationAsync(AuthenticationProperties properties, TokenResponse tokens)
+        protected virtual async Task<AuthenticationTicket> CreateTicketAsync(AuthenticationProperties properties, TokenResponse tokens)
         {
-            var context = new OAuthGetUserInformationContext(Context, Options, Backchannel, tokens)
+            var identity = new ClaimsIdentity(Options.ClaimsIssuer);
```

Comment:
IMHO, `AuthenticationScheme` definitely sounds more appropriate, @Tratcher thinks we should use `Options.ClaimsIssuer` instead.